lib/content-services/src/lib/content-node-share/content-node-share.dialog.spec.ts

```diff
@@ -45,6 +45,13 @@ describe('ShareDialogComponent', () => {
     let component: ShareDialogComponent;
     let appConfigService: AppConfigService;
 
+    const getShareToggleId = '[data-automation-id="adf-share-toggle"]';
```

Comment:
Sorry, I did not notice it earlier but can you rename it to something what does not contain 'get' word? Get is verb so it suggest like it is function but it is constant